replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -107,8 +107,8 @@ public ReplicatorConfigImpl() {
   }
 
   @Override
-  public void fromMap(Map<String, Object> properties) {
-    super.fromMap(properties);
+  public int fromMap(Map<String, Object> properties) {
+    int propVersion = super.fromMap(properties);
```

Comment:
✏️ How about renaming it `currentVersion` or `serializedVersion`?

Reply:
Changed to serializedVersion